companion/src/generaledit.cpp

```diff
@@ -13,53 +13,43 @@
 #define BEEP_VAL_SHIFT   3
 
 GeneralEdit::GeneralEdit(RadioData &radioData, QWidget *parent) :
-    QDialog(parent),
```

Comment:
yep, thought about that later on... really sorry! learning to develop in a team....